lib/couch_tap/changes.rb

```diff
@@ -141,19 +147,26 @@ def find_or_create_sequence_number
     end
 
     def update_sequence(seq)
-      database[:couch_sequence].where(:name => source.name).update(:seq => seq)
+      data = {
+        name: source.name,
+        seq: seq,
+        updated_at: DateTime.now
+      }
+      database[:couch_sequence]
+        .insert_conflict(target: :name, update: data)
+        .insert(data.merge(created_at: data[:updated_at]))
       self.seq = seq
     end
 
     def create_sequence_table
       database.create_table :couch_sequence do
         String :name, :primary_key => true
-        Bignum :seq, :default => 0
+        String :seq, :default => 0
```

Comment:
In CouchDb 1.6, seq was a number, in CouchDb 2.0 seq is a convoluted hashwannabethingmagigmegastring: `42-g1AAAACbeJzLYWBgYMpgTmEQTM4vTc5ISXLIyU9OzMnILy7JAUklMiTV____PyuDOVEoFyjAbphkbGFuaIJNAx5j8liAJEMDkPoPNU0CbFqKZbKZmYElNn1ZAFfbMG8`